cliquet/storage/postgresql/client.py

```diff
@@ -17,41 +21,44 @@ def __init__(self, engine):
         #                                   loads=json.loads)
 
     @contextlib.contextmanager
-    def connect(self, readonly=False):
-        """Pulls a connection from the pool when context is entered and
+    def connect(self, readonly=False, force_commit=False):
+        """
+        Pulls a connection from the pool when context is entered and
         returns it when context is exited.
 
         A COMMIT is performed on the current transaction if everything went
         well. Otherwise transaction is ROLLBACK, and everything cleaned up.
-
-        XXX: Committing should not happen here but using a global transaction
-        manager like `pyramid-tm`.
         """
-        connection = None
-        trans = None
+        with_transaction = (not readonly and self.commit_manually)
```

Comment:
My reference is always the (recent) Python stdlib for style. It's not considered "pythonic" to have superfluous characters.  The typical example is for tuple.

```python
one = two, three, four     # do this
one = (two, three, four)     # not this
```
One exception of avoiding \ to break a line.

e.g. do:

```python
import (one, two,
            three)     # yes!!
import one, two, \
           three   # boooo!
```

So I guess my question is: what's the benefit for our code base to do this differently ?  If none, I am +1 to use Python stdlib conventions. (We should check of course that it's wlays done like that there :))